### ISSUE TITLE
Buymeacoffee - Having issues with donating

### ISSUE THREAD

**Issue by Ecoleaf1** [jeffshee/hidamari#60]
I am aware that this has nothing to do with the program itself, however I didn't know where else to forward this issue.

I tried donating by using the BuyMeACoffee link, however an error prompt appeared stating "Your account cannot currently make live charges.".

I have called my bank to make sure my card wasn't declined, however there is no history of me attempting to donate via the BuyMeACoffee URL. Perhaps the payment is configured incorrectly?

Cheers

![Error buymeacoffee](https://user-images.githubusercontent.com/51735180/184571107-96084af8-a27d-4afd-ba1e-d77a5a4524f9.jpg)

**Comment by jeffshee** [comment]
Hi @Ecoleaf1, thanks for reaching out to me. And also thanks for your support!

I don't have much experience with _buymeacoffee_. It seems there is some problem in my setting and I had tried to change that.
Not sure if this fixes the problem or not, because when I try to do a test transaction, I got my card declined every time.

Maybe I should try another service if _buymeacoffee_ just doesn't work for me.
